src/main/java/com/currencycloud/client/CurrencyCloudClient.java

```diff
@@ -922,11 +923,19 @@ public Payment createPayment(Payment payment, @Nullable Payer payer) throws Curr
         );
     }
 
+    public PaymentAuthorisations authorisePayment(List<Payment> payments) throws CurrencyCloudException {
+         return api.authorisePayment(
+                authToken,
+                userAgent,
+                payments.stream().map(p -> p.getId()).collect(Collectors.toList())
```

Comment:
Curious as to why use Collectors here

Reply:
??? How else would you get a list of ids?

Reply:
Didn't mean Collectors.toList() specifically but rather the Stream/Collectors approach instead of String[] . It's not wrong, quite the contrary, but it won't be consistent with the rest of the SDK. 

I think we need to plan a thorough refactor ahead of release version 3 of the API. Will catch up tomorrow.

Reply:
What do you mean? The other usages are a List and Collections.